core/client/fs/src/main/java/alluxio/client/file/AlluxioEvent.java

```diff
@@ -0,0 +1,78 @@
+package alluxio.client.file;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Defines and Manages well-known Alluxio events.
+ *
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ * !!! - Keep increasing the event-id for new events. !!!
+ * !!! - Don't reassign event ids.                    !!!
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ */
+public enum AlluxioEvent {
+  // Alluxio master events.
+  Master_Process_Starting(1, EventType.INFO, "Master process starting."),
+  Master_Process_Stopping(2, EventType.INFO, "Master process stopping."),
+  // Alluxio worker events.
+  Worker_Process_Starting(3, EventType.INFO, "Worker process started."),
+  Worker_Process_Stopping(4, EventType.INFO, "Worker process stopping.");
+
+  // logger.
+  private static final Logger LOG = LoggerFactory.getLogger(AlluxioEvent.class);
+
+  /** The unique event id. */
+  private final int mId;
```

Comment:
What is the id for? Wouldn't the name of the enum be enough?

Reply:
Numeric IDs are easier to communicate imo. So that's why I took the trouble to define it.